### PR TITLE
fix(ci/cd): pin flutter version disable score 

### PR DESCRIPTION
**Pull request by gagantrivedi**

### CODE REVIEW

File: .github/workflows/publish_master.yml [hunk 1]
```diff
@@ -8,21 +8,25 @@ jobs:
...
-  score:
-    runs-on: ubuntu-latest
-    name: Dart package score
 
-    steps:
-      - uses: actions/checkout@v2
-      - uses: axel-op/dart-package-analyzer@v3
-        with:
-          githubToken: ${{ secrets.GITHUB_TOKEN }}
+  # TODO enable this after upgrading to dart 3.x
```
Comment (khvn26) [inline .github/workflows/publish_master.yml:20]: Is the upgrade planned soon?

Comment (gagantrivedi) [reply .github/workflows/publish_master.yml:20]: Not, really no

Comment (khvn26) [reply .github/workflows/publish_master.yml:20]: I'd prefer to remove the code that's commented out, create an issue for the upgrade, and reference the created issue in the TODO.

Comment (gagantrivedi) [reply .github/workflows/publish_master.yml:20]: Yes, good shout

**Review by matthewelwell — APPROVED** [review]

**Pull request merged**